src/classes/BDI_DataImportDeleteBTN_CTRL.cls

```diff
@@ -41,23 +41,50 @@ public with sharing class BDI_DataImportDeleteBTN_CTRL {
 
     /** @description Action type specified to delete all Data Import records */
     public static final String ACTION_DELETE_ALL = 'deleteAll';
-    
+
+    /** @description The list of found Data Import records to be deleted */
+    @TestVisible
+    private List<DataImport__c> dataImports;
+
+    /** @description True when user can delete data import records */
+    @TestVisible
```

Comment:
@TestVisible and public are redundant here. Should this be private?

Could we simplify this as `public static final Boolean canDelete = SObjectType.DataImport__c.isDeletable();` ?

Reply:
@lmeerkatz - Thanks for the review! It is a private set, but this allows the test to set it to validate what happens when a user does not have permissions without having to muck around with users/profiles in the test.